editor/plugins/tiles/tile_set_atlas_source_editor.cpp

```diff
@@ -2195,55 +2195,70 @@ void TileSetAtlasSourceEditor::edit(Ref<TileSet> p_tile_set, TileSetAtlasSource
 	_update_current_tile_data_editor();
 }
 
-void TileSetAtlasSourceEditor::init_source() {
+void TileSetAtlasSourceEditor::init_source(Vector<Ref<TileSetAtlasSource>> newSources) {
+	hackBodge = newSources;
```

Comment:
`hackBodge` is used to hold the new sources while waiting for the user to close the dialog.

Reply:
Renamed to `pendingAtlases`